src/main/java/com/google/sps/servlets/utils/CommonWordsRetriever.java

```diff
@@ -14,18 +14,33 @@
 
 package com.google.sps.servlets.utils;
 
+import java.io.File;
+import java.io.FileNotFoundException;
 import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.List;
+import java.util.Scanner;
 
-/** Class provides commonly used words that should be ignored within the word map */
+/** Class that provides commonly used words that should be ignored within the word map */
 public class CommonWordsRetriever {
-  // TODO: Parse text file for common words
-  private static final List<String> commonWordsList =
-      new ArrayList<>(
-          Arrays.asList(
-              "the", "it", "at", "and", "to", "we", "can", "are", "of", "is", "am", "i", "this",
-              "was"));
+  private static final List<String> commonWordsList = populateWordList();
+
+  /** Populate a list of strings from a text file containing the words to ignore in the word map */
+  private static List<String> populateWordList() {
+    List<String> commonWords = new ArrayList<String>();
+    try {
+      File file =
+          new File(
+              System.getProperty("user.dir")
+                  + "/src/main/java/com/google/sps/servlets/utils/resources/common_words.txt");
```

Comment:
Can we declare this String as a `private static final String` constant at the top of the class?

Generally it's best to declare all constants in this way.